### PR TITLE
Add readOnly option #262

### PR DESCRIPTION
**Pull request by sebastienwarin**

### CODE REVIEW

**Comment by jcubic** [comment]
What is the difference between readOnly and enabled right now they look the same to me. And if you don't want to to enable terminal on click then you can freeze the terminal.

**Comment by sebastienwarin** [comment]
Sure ! I have remove this option and fix the frozen mode

File: js/jquery.terminal-0.11.11.js [hunk 1]
```diff
@@ -5220,7 +5196,7 @@
                     $(window).unbind('mousemove');
                     if (!wasDragging && ++count == 1) {
                         count = 0;
-                        if (!settings.readOnly && !self.enabled()) {
+						if (!self.enabled() && !frozen) {
```
Comment (jcubic) [inline js/jquery.terminal-0.11.11.js]: Please use 4 spaces instead of tabs.

File: js/jquery.terminal-src.js [hunk 2]
```diff
@@ -5219,8 +5195,8 @@
                     isDragging = false;
                     $(window).unbind('mousemove');
                     if (!wasDragging && ++count == 1) {
-                        count = 0;
-                        if (!settings.readOnly && !self.enabled()) {
+                        count = 0;
```
Comment (jcubic) [inline js/jquery.terminal-src.js]: Please remove space after semicolon.

**Pull request merged**